### PR TITLE
Add ESP8285 900MHz TX Target

### PR DESCRIPTION
**Pull request by ot0tot**
Add ESP8285 900MHz TX target, in conjunction with https://github.com/ExpressLRS/ExpressLRS/pull/2566

### CODE REVIEW

File: targets.json [hunk 1]
```diff
@@ -794,6 +794,15 @@
                 "firmware": "Unified_ESP32_900_TX",
                 "prior_target_name": "Unified_ESP32_900_TX"
             }
```
Comment (pkendall64) [inline targets.json:796]: You need a comma on the end of this line

Comment (ot0tot) [reply targets.json]: Good catch, fixed.

**Comment by Oborysenko** [comment]
@pkendall64 , Can you revoke this PR. It's contain wrong JSON.
The JSON you provided seems to be incorrect due to a trailing comma after the last key-value pair. JSON does not allow trailing commas in objects. You should remove the comma after "firmware": "Unified_ESP8285_900_TX".

**Comment by CapnBry** [comment]
Haha everything about this PR is wrong! @ot0tot you're on my naughty list.

**Pull request merged**